src/utils/media-utils.js

```diff
@@ -615,7 +621,7 @@ export async function resolveMediaInfo(urlString) {
 
   // We want to resolve and proxy some hubs urls, like rooms and scene links,
   // but want to avoid proxying assets in order for this to work in dev environments
-  const isLocalAsset = isNonCorsProxyDomain(url.hostname);
+  const isLocalAsset = isNonCorsProxyDomain(url.hostname) && !(await isHubsDestinationUrl) && !(await isHubsAvatarUrl);
```

Comment:
I think `isHubsDestinationUrl` and `isHubsAvatarUrl` are functions. Is `!(await isHubsDestinationUrl) && !(await isHubsAvatarUrl);` intentional? Or you really wanted `!(await isHubsDestinationUrl(url.href)) && !(await isHubsAvatarUrl(url.href));`?